.editorconfig

```diff
@@ -25,16 +25,22 @@ dotnet_naming_rule.private_members_with_underscore.severity = suggestion
 
 dotnet_naming_symbols.private_fields.applicable_kinds           = field
 dotnet_naming_symbols.private_fields.applicable_accessibilities = private
+# dotnet_naming_symbols.private_fields.required_modifiers = abstract,async,readonly,static # all except const
```

Comment:
Commented this out because it causes fields to not be prefixed with underscores and creating new fields from ctor params results in `this.myProperty` instead of `_myProperty`

Reply:
Damn, worked in Rider - But I guess that's because the other things are default conventions